proto/proto/penumbra/client/v1alpha1/client.proto

```diff
@@ -195,3 +206,74 @@ message PrefixValueResponse {
   string key = 1;
   bytes value = 2;
 }
+
+// Defines the gRPC query service for proxying requests to an upstream Tendermint RPC.
+service TendermintProxyService {
+  // Status queries the current status.
+  rpc GetStatus(GetStatusRequest) returns (GetStatusResponse) {
+    option (google.api.http).get = "/status";
+  }
+
+  // Broadcast a transaction asynchronously.
+  rpc BroadcastTxAsync(BroadcastTxAsyncRequest) returns (BroadcastTxAsyncResponse) {
+    option (google.api.http).get = "/broadcast_tx_async";
+  }
+
+  // Broadcast a transaction synchronously.
+  rpc BroadcastTxSync(BroadcastTxSyncRequest) returns (BroadcastTxSyncResponse) {
+    option (google.api.http).get = "/broadcast_tx_sync";
+  }
+}
```

Comment:
Did we make this RPC up, or did we vendor / adapt it from somewhere? (Not sure it really matters, just thought it might be good to note which case we were in)

Reply:
This is one that we made up, since the existing cosmos tendermint gRPC client didn't have these methods